releng/com.espressif.idf.target/com.espressif.idf.target.target

```diff
@@ -122,5 +122,13 @@
 			<unit id="org.eclipse.embedcdt.feature.group" version="6.4.0.202307251916"/>
 			<unit id="org.eclipse.embedcdt.packs.feature.group" version="6.4.0.202307251916"/>
 		</location>
+		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+			<repository location="http://download.eclipse.org/linuxtools/update"/>
+			<unit id="org.eclipse.linuxtools.dataviewers.feature.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.linuxtools.gprof.feature.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.linuxtools.gcov.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.linuxtools.profiling.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.linuxtools.profiling.remote.feature.group" version="0.0.0"/>
```

Comment:
Can you check if we need all of them? Probably, we need only org.eclipse.linuxtools.gcov.feature.group